package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "functional-ttt",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
```

Comment:
I forget what eject does; not sure if that gets included in normal react apps; would be good to hear what the other mentors do.

Reply:
So, create-react-app helps bootstrap a bunch of configurations.  You'd `eject` if you no longer want use the defaults and don't want create-react-app to manage the dependencies/configurations for you. 

From [FreeCodeCamp](https://www.freecodecamp.org/news/create-react-app-npm-scripts-explained/): 
> The last command, eject, is used to remove the dependency on react-scripts and expose the build tools and configurations for you to modify.

> All the configuration files from react-scripts will be copied into your project root's config/ folder, and the scripts to run the build will be copied into the scripts/ folder. The dependencies will also be moved into your root's package.json file.

> This command is a one-way operation. Once you have ejected from CRA set up, you can't undo it.

I personally wouldn't mind leaving it in there so you know it's an option that is specific to create-react-app. It'd get deleted if you end up actually ejecting it in the future in order to use your own configs.

Reply:
thanks Claire!

Reply:
Thanks! I'll keep it in the ReadMe then!